bin/install.sh

```diff
@@ -341,6 +346,20 @@ echo WEB_CONCURRENCY=$webConcurrency >> $envFile
 
 echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout ready shortly."
 
+######################################################
+# Store off environment configuration in /etc/waffle #
+######################################################
+echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle/ca-certificates
+sudo cp $envFile /etc/waffle/environment.list
```

Comment:
To clarify, leaving `./environment.list` around as it is now allows the install to run multiple times without having to answer the hard question above.